other/containers/tilck-ci-build/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Also tried to use debian/stable-slim as base, but image size went up compared
+# to ubuntu 20.04
+FROM ubuntu:20.04
+# We don't want any of the interactive features of apt, so we just disable them.
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt update && apt upgrade -y && \
+apt install git-core gcc g++ cmake imagemagick clang gcc-multilib g++-multilib \
+python3-minimal apt gawk unzip parted wget qemu-system-x86 -y && \
+mkdir -p tmp-tilck && \
+mkdir -p tc/toolchain2 && \
+git clone https://github.com/vvaltchev/tilck.git tmp-tilck/tilck && \
+tmp-tilck/tilck/scripts/build_toolchain && \
+tmp-tilck/tilck/scripts/build_toolchain -s gnuefi mtools zlib \
+busybox gmock gmock_arch gtest gtest_arch lcov libmusl lua micropython ncurses \
+tcc vim gcc_glibc_toolchain && \
+cp -r tmp-tilck/tilck/toolchain2/ tc && \
+rm -rf tmp-tilck && \
+mkdir -p tmp-tilck/tilck && \
+ln -s /tc/toolchain2 /tmp-tilck/tilck/toolchain2
```

Comment:
Sorry, but this is total non-sense, man: why deleting tmp-tilck and then symlinking /tc/toolchain2 there? Even if that made everything to work at the end, is still wrong. I proposed something COMPLETELY different. Let me repeat that in more explicit way:

1. git clone tilck
2. mkdir /tc/toolchain2
3. ln -s /tc/toolchain2 /tilck-tmp/tilck/toolchain2
4. ./tilck-tmp/tilck/build_toolchain
5. all the other steps
6. rm -rf /tmp-tilck

Still, I realize now that even that **might not work** if during the build libmusl realpath() is not used, so **exactly** the old absolute paths are expected to exist. Therefore, let's use the official way supported by Tilck for an out-of-tree toolchain: 

1.  mkdir /tc
2. export TCROOT_PARENT="/tc"
3. git clone https://..... tilck-tmp # NOTE: no need for /tilck-tmp/tilck. Clone directly in /tilck-tmp. It's nicer.
4. ./tilck-tmp/build_toolchain
5. everything else here
6. rm -rf /tilck-tmp

No fancy symlinks, nothing. By just exporting the right environment variable, you can tell exactly where the toolchain2 directory have to be placed.

Reply:
I thought what i did was the correct way of doing it, i'm sorry.